src/functions-and-arrays.js

```diff
@@ -1,19 +1,48 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(num1, num2) {
+  if(num1>num2){
+    return num1
+  } else {
+    return num2
+  }
+}
+console.log(maxOfTwoNumbers(4,5));
+
 
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
 
+function findLongestWord(arr) {
```

Comment:
The logic here is perfect, but it is not passing the tests, because your parameter is called arr, so you need to do all the code related to that, for in your if statement if needs to compare 
```
    if (arr[i].length > longestWord.length) {
      longestWord = arr[i]
        }
```

instead of 

```
     if (words[i].length > longestWord.length) {
      longestWord = words[i]
```